webpack/components/NewTemplateSync/components/NewTemplateSyncForm/__tests__/__snapshots__/NewTemplateSyncForm.test.js.snap

```diff
@@ -1,11 +1,116 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`NewTemplateSyncForm should render for export settings 1`] = `
-<Form
-  disabled={false}
-  errorTitle="Warning! "
+<ForemanForm
```

Comment:
IMO this test/snapshot falls into the definition of a **False positive** and **False negative**.

Most of the data in the snapshot look irrelevant while the only relevant field is the `initialValues`.